Knossos.NET/ViewModels/Templates/TaskItemViewModel.cs

```diff
@@ -204,6 +204,220 @@ public TaskItemViewModel()
+                                    if (cancellationTokenSource.IsCancellationRequested)
+                                        throw new TaskCanceledException();
+                                }
+
+                                //we can know if the verify completed by checking the copySrcList
+                                if(copySrcList.Any() && copySrcList.Count() == copyDstList.Count())
+                                {
+                                    ProgressBarMax = copySrcList.Count();
+                                    //Copy files
+                                    for (int i = 0; i < copySrcList.Count(); i++)
+                                    {
+                                        //Make sure the dest folder structure exist
+                                        Directory.CreateDirectory(Path.GetDirectoryName(copyDstList[i])!);
+                                        //First lets try to hardlink, if one fails or it is disabled, revert to copy files
+                                        if (useHardlinks)
+                                        {
+                                            Info = "Hardlink file: " + (i + 1).ToString() + " / " + copySrcList.Count() + " (" + Path.GetFileName(copyDstList[i]) + ")";
+                                            useHardlinks = HardLink.CreateFileLink(copySrcList[i], copyDstList[i]);
+                                        }
+                                        if (!useHardlinks)
+                                        { 
+                                            Info = "Copy file: " + (i + 1).ToString() + " / " + copySrcList.Count() + " (" + Path.GetFileName(copyDstList[i]) + ")";
+                                            File.Copy(copySrcList[i], copyDstList[i], true);
+                                        }
+                                        ++ProgressCurrent;
+                                        if (cancellationTokenSource.IsCancellationRequested)
+                                            throw new TaskCanceledException();
+                                    }
+                                    //If we get here whiout any exceptions it means it completed successfully
+                                    Log.Add(Log.LogSeverity.Information, "TaskItemViewModel.TryToCopyFilesFromOldVersions()", "All files needed for nebula file "+ file.filename +" were copied from "+ oldVer+ ". Download from nebula was skipped successfully.");
+                                    if (!useHardlinks)
+                                    {
+                                        Info = copySrcList.Count() + " files copied OK";
+                                    }
+                                    else
+                                    {
+                                        Info = copySrcList.Count() + " files hard-linked OK";
+                                    }
+                                    //IsCompleted = true;
```

Comment:
Should this commented out line stay?

Reply:
The only thing this will do is show the "completed" in green below the task that dosent make much sense here because it is already display a "x files copied ok".